index.js

```diff
@@ -18,4 +20,339 @@ const openMenu = document.querySelector('.button');
+    newarr[i].image = image.innerHTML;
+    newarr[i].technologies = document.querySelector('#pro-list');
+    newarr[i].link = link.innerHTML;
+    newarr[i].source = source.innerHTML;
+    arr.push(newarr[i]);
+  } else if (i == 2) {
+    newarr[i].name = document.querySelector('#prohead1');
+    newarr[i].description = document.querySelector('#propara1');
+    newarr[i].image = image2.innerHTML;
+    newarr[i].technologies = document.querySelector('#card');
+    newarr[i].link = link.innerHTML;
+    newarr[i].source = source.innerHTML;
+    arr.push(newarr[i]);
+  } else if (i == 3) {
+    newarr[i].name = document.querySelector('#prohead2');
+    newarr[i].description = document.querySelector('#propara2');
+    newarr[i].image = image3.innerHTML;
+    newarr[i].technologies = document.querySelector('#pro-list2');
+    newarr[i].link = link.innerHTML;
+    newarr[i].source = source.innerHTML;
+    arr.push(newarr[i]);
+  } else if (i == 4) {
+    newarr[i].name = document.querySelector('#prohead3');
+    newarr[i].description = document.querySelector('#propara3');
+    newarr[i].image = image4.innerHTML;
+    newarr[i].technologies = document.querySelector('#pro-list3');
+    newarr[i].link = link.innerHTML;
+    newarr[i].source = source.innerHTML;
+    arr.push(newarr[i]);
+  } else if (i == 5) {
+    newarr[i].name = document.querySelector('#prohead4');
+    newarr[i].description = document.querySelector('#propara4');
+    newarr[i].image = image5.innerHTML;
+    newarr[i].technologies = document.querySelector('#pro-list4');
+    newarr[i].link = link.innerHTML;
+    newarr[i].source = source.innerHTML;
+    arr.push(newarr[i]);
+  } else if (i == 6) {
+    newarr[i].name = document.querySelector('#prohead5');
+    newarr[i].description = document.querySelector('#propara5');
+    newarr[i].image = image6.innerHTML;
+    newarr[i].technologies = document.querySelector('#pro-list5');
+    newarr[i].link = link.innerHTML;
+    newarr[i].source = source.innerHTML;
+    arr.push(newarr[i]);
+  }
+}
+
+const section = document.createElement('section');
+section.id = 'content';
+const div = document.createElement('div');
+div.id = 'headingpop';
+const target = document.querySelector('#wrap');
+target.insertBefore(section, target.childNodes[2]);
+document.body.querySelector('#content').appendChild(div);
+const closebutton = document.createElement('div');
+closebutton.innerHTML = '<button id="closepopup"><img src="./mobilebutton/close.png" alt="socail-link"></button>';
+document.body.querySelector('#content').appendChild(closebutton);
+
+const button0 = document.querySelector('#project-button');
+const button1 = document.querySelector('#probutton');
+const button2 = document.querySelector('#probutton2');
+const button3 = document.querySelector('#probutton3');
+const button4 = document.querySelector('#probutton4');
+const button5 = document.querySelector('#probutton5');
+const button6 = document.querySelector('#probutton6');
```

Comment:
- **[OPTIONAL]** Kindly refactor your code to avoid repetitive code, remember DRY (don't repeat yourself) for the better practices of JavaScript. 👍🏽